cpp/main.cpp

```diff
@@ -0,0 +1,45 @@
+#include <iostream>
```

Comment:
@deadlyicon I'd double check with @lumodon, but I think this include is necessary for the print statements at the end.